addon/adapters/twitch-channel-feed.js

```diff
@@ -0,0 +1,8 @@
+import TwitchAdapter from 'ember-data-twitch/adapters/twitch';
+
+export default TwitchAdapter.extend({
+
+  pathForType(/* modelName */) {
+    return 'feed';
```

Comment:
Is this basically to handle instances where we don't want to pluralize? We can probably generalize this even more by having the root adapter have a `pluralize` boolean that defaults to true, and we can just set these instances to `false` to avoid having to completely override `pathForType` on every adapter that exhibits this pattern. If it's a common pattern, definitely makes sense to handle it in the root adapter.